admin/build-doc

```diff
@@ -20,7 +20,10 @@ if command -v dpkg >/dev/null; then
         exit 1
     fi
 elif command -v yum >/dev/null; then
-    for package in python-devel python-pip python-virtualenv doxygen ditaa ant libxml2-devel libxslt-devel Cython graphviz; do
+    if ! command -v pip >/dev/null; then
+        missing="${missing:+$missing }python-pip"
```

Comment:
@bellaalleb i think #19751 is a better fix. as it works for EL-7 also.

Reply:
actually, when you exec "yum install python-pip", it would automatically install pip match with your python version.
 #19751 didn't work include python34-pip. Though I'm not sure if ceph works fine with python3, there are no requirements about python version in official doc, thus I think we should support python3 as well.

Reply:
> it would automatically install pip match with your python version

that's not true.

```
bash-4.2$ rpm -q --provides python2-pip
python-pip = 8.1.2-5.el7
python2-pip = 8.1.2-5.el7
bash-4.2$ rpm -q --provides python34-pip
python34-pip = 8.1.2-5.el7
```

> #19751 didn't work include python34-pip.

i am not sure i follow you. #19751 works in the sense that it builds the doc on EL7. but, yeah, it does not install "python34-pip". because i don't think python34 is the default python on EL7. and `virtualenv` just uses the default python.

> thus I think we should support python3 as well.

agreed. could you be more specific? what problem do you run into when using py3 to build the doc?

Reply:
> Here I just think if someone change default python to python34 on EL7, we may support that too.

i don't think this will happen in EL7.